platforms/android/app/src/main/res/values/strings.xml

```diff
@@ -3,9 +3,14 @@
     <string name="app_name">सीसीआई - CCI</string>
     <string name="launcher_name">@string/app_name</string>
     <string name="activity_name">@string/launcher_name</string>
-    <string name="content_extraction">यह प्रक्रिया केवल एक बार की जाएगी। इसमें १० मिनट तक का समय लग सकता है।</string>
+    <string name="content_extraction_hindi">यह प्रक्रिया केवल एक बार की जाएगी। इसमें १० मिनट तक का समय लग सकता है।</string>
+    <string name="content_extraction_english">This process will be done only once. It can take up to 10 minutes.</string>
     <string name="splash_screen">Splash Screen</string>
     <string name="startPercent">0 %</string>
     <string name="mainFileVersion">mainFileVersion</string>
     <string name="patchFileVersion">patchFileVersion</string>
+    <string name="dataPath">dataPath</string>
```

Comment:
You have used \n to separate in English and Hindi in SD card but have used separate strings in for extraction screen. Can you make it consistent with \n?

Reply:
Using different textView in the extraction screen gives more flexibility for positioning. With '\n' the vertical spacing will not be achieved.